.gitignore

```diff
@@ -3,6 +3,7 @@
 *.sqlite
 *.sublime*
 *\.*swp
+.idea
```

Comment:
@m3brown i put this here incase anyone develops code using jetbrain IDE's then it will ignore unnecessary ide specific files